src/cdk/coercion/string-array.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * Coerces a value to an array of trimmed non-empty strings.
```

Comment:
It would be good to expand this description to give more details about the function's behavior. E.g., mentioning now `null` and `undefined` are treated, mentioning that objects will be `toString`ed, etc.

Reply:
Updated the docs with a better description (including describing some special cases) and the params.